paradise_papers_search/fetch_api/views.py

```diff
@@ -10,7 +10,7 @@ def get(self, request):
             'country': request.GET.get('c', ''),
             'jurisdiction': bool(request.GET.get('j', False)),
```

Comment:
Why jurisdiction is a bool and country isn't ? The way they work are pretty much the same.   Thanks